.github/workflows/prepare-release.yml

```
@@ -0,0 +1,60 @@
+name: Prepare release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to release, e.g. 1.30.0'
+        required: true
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Validate version
+        run: |
+          version=${{ inputs.version }}
+          if [[ ! "$version" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "unexpected version: $version"
+            exit 1
+          fi
+
+      - name: Use CLA approved github bot
+        run: .github/workflows/scripts/use-cla-approved-github-bot.sh
+
+      - name: Update schema files
+        run: |
+          if ! grep -q "^  next:$" schema-next.yaml; then
+            echo "String 'next:' not found in the file"
+            exit 1
+          fi
+
+          version=${{ inputs.version }}
+          sed -i "0,/^  next:$/s//  $version:/" schema-next.yaml
+          cp schema-next.yaml "schemas/$version"
+          git add "schemas/$version"
+
+          sed -i "0,/^  $version:$/s//  next:\n  $version:/" schema-next.yaml
+
+      - name: Update change log
+        run: |
+          make chlog-update VERSION=${{ inputs.version }}
```

nit: I think it should be 

```suggestion
          make chlog-update VERSION=v${{ inputs.version }}
```

at least that's what we have in the changelog